server.js

```diff
@@ -14,16 +14,85 @@ import cors from "cors";
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
-
+const listEndpoints = require('express-list-endpoints')
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
+
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get("/", (request, response) => {
+  response.json(listEndpoints(app));
+})
+
+//Show all data
+app.get("/fullmoon", (request, response) => {
+  const fullmoon = fullMoonData;
+  if (fullmoon) {
+    response.status(200).json({
+      success: true,
+      message: "ok",
+      body: {
+        fullMoonData: fullmoon
+      }
+    });
+  } else {
+    response.status(500).json({
+      success: false,
+      message: "Something went wrong",
+      body: {}
+    });
+  } 
+})
+
+//Endpoint for one specific date
+app.get("/fullmoon/date/:date", (request, response) => {
+  const singleFullmoon = fullMoonData.find((fullmoon) => {
+    const fullmoonDate = new Date(fullmoon.Date);
+    const requestDate = new Date(request.params.date);
+    return fullmoonDate.getTime() === requestDate.getTime();
+  });
+
+  if (singleFullmoon) {
+    response.status(200).json({
+      success: true,
+      message: "ok",
+      body: {
+        fullmoon: singleFullmoon
+      }
+    });
+  } else {
+    response.status(404).json({
+      success: false,
+      message: "No fullmoon this date",
+      body: {}
+    });
+  } 
 });
 
+//Endpoint for different weekdays
+app.get("/fullmoon/weekday/:weekday", (request, response) => {
+  const weekday = request.params.weekday.toLowerCase();
+  const weekdayWithFullmoon = fullMoonData.filter((day) => day.Day.toLocaleLowerCase() === weekday)
```

Comment:
Maybe use toLowerCase here as well so it's consistent throughout the project.

Good to use filter here to return several results